dump/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
maybe this is a node thing, but what is the dump directory for?

Reply:
When a submit fails for some reason, it saves the request data here.